docs/usage.md

```diff
@@ -3,130 +3,441 @@
 
-Post-review, you can connect the AI generated playbook blocks to the main playbook flow and create a complete workflow.
+### Use Case Example: Blocking a Malicious IP Address
 
-As  demonstrated, this represents a significant advancement in playbook design and is of enormous  assistance to individuals who are just starting out developing playbooks.
+A common use case is blocking a malicious IP address that has been identified as a threat. For example, if an IP address is flagged as malicious, blocking it prevents further interactions with the threat, protecting your network from potential harm.
 
-#### Common use cases for generating playbook steps using FortiAI
+### Blocking Indicators Using FortiAI Assistant
 
-Some other playbook generation use cases that you could use to familiarize yourself with FortiAI are:
+1. **Initiate Blocking Action**:
+   - Issue a command to block the indicator using the [configured connector]. For example:
 
-- Extract IOCs from CSV and Create the Indicator Record for each
-- Extract Indicators from Alert Source Data and Create Indicator Records for each
-- Fetch Unread Emails from Exchange and create Alert record
-- Notify using Slack when Alert severity changes
-- On Create of alert add a comment *Hello, Analyst!* on Alert
-- Check the IP's reputation using VirusTotal.
-    - If it's considered malicious, set the indicator reputation as malicious.
-    - If it's considered suspicious, set the indicator reputation as suspicious.
-    - Else, set the indicator reputation as good
-- Submit a file for analysis to FortiSandbox and obtain the verdict for the analysis job
-- On Delete of an alert, using Exchange send an email to Admin
-- Find the reputation of the indicator using VirusTotal and prompt the user for action: either block it using FortiGate or take no action.
-- Manually trigger a playbook for an Alert Record to change the status to Investigating.
+    >*Block IP address `[IP-ADDRESS]` using the [configured connector]*
 
-### Ask a contextual question to FortiAI
+   - The assistant uses the specified connector to block the indicator.
+
+2. **Provide Indicator Details**:
+   - Specify the details of the indicator to be blocked. For example:
+
+    >*Block domain `[DOMAIN]` due to malicious activity*
+
+   - The assistant processes the request and blocks the indicator accordingly.
+
+3. **Review Blocking Status**:
+   - Verify that the indicator has been successfully blocked. Check the status or confirmation provided by the assistant to ensure the action was completed.
+
+### Review Blocking Actions
+
+- Confirm that the blocking action has been executed correctly and that the indicator is no longer active. Ensure that the blocking process has addressed the threat effectively.
+
+### Integrate Blocking Actions into Workflow
```

Comment:
Again review and Integrate Blocking Actions into Workflow are repeated. Please remove them.

Reply:
Done